simple/stats/nl.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2023 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+import pandas as pd
+from stats.data import StatVar
+from util.filehandler import FileHandler
+
+_MODEL_NAME = "all-MiniLM-L6-v2"
+_DCID_COL = "dcid"
+_SENTENCE_COL = "sentence"
+
+
+def generate_sv_sentences(svs: list[StatVar], sentences_fh: FileHandler):
+  """Generates sentences based on the name, description and NL sentences of the specified SVs.
+
+    The SV dcids and sentences are written to a CSV using the specified FileHandler
+    """
+  rows = []
```

Comment:
nit: if this is the only reason to import pandas, perhaps use csv.DictWriter instead?

Reply:
Pandas is used at other places in the package so using it here as well.